pkg/secrethub/client.go

```diff
@@ -72,7 +80,7 @@ type Client struct {
 	// These are cached
 	repoIndexKeys map[api.RepoPath]*crypto.SymmetricKey
 
-	appInfo   *AppInfo
+	appInfo   []*AppInfo
```

Comment:
I think this can just as well be a `[]AppInfo` -> looks cleaner.

Reply:
Hmm, but to make that nice and consistent we'd have to change (and break) `WithAppInfo` too, because that takes `*AppInfo`. So what do we want here?

Reply:
Ugh. Yuck. In that case, just leave it.